sites/friday/src/add-services/clickhouse.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "ClickHouse"
+weight: -110
+description: 
+---
+
+ClickHouse is a high-performance column-oriented, distributed, OLAP (Online Analytical Processing) database.</br>
+It allows you to generate real-time analytical data reports using SQL queries.</br>
+For more information, see the [ClickHouse documentation](https://ClickHouse.com/docs).
+
+{{% note %}} 
+
+{{% vendor/name %}} supports ClickHouse with the following limitations:
+
+- High availibility of service isn't supported.
+- You can only configure single-node ClickHouse clusters.
+
+{{% /note %}} 
+
+## Supported versions
+
+- 23
```

Comment:
This version exposed to user should be more precise as I mentioned here https://platformsh.slack.com/archives/C8DNLHHL7/p1708016189938389?thread_ts=1707145449.371169&cid=C8DNLHHL7
It's not blocking for releasing this